webapp/src/js/controllers/contacts-edit.js

```diff
@@ -176,7 +178,7 @@ angular.module('inboxControllers').controller('ContactsEditCtrl',
       });
 
     $scope.save = function() {
-      if (ctrl.enketoStatus.saving) {
+      if (ctrl.enketoSaving) {
```

Comment:
This raised an interesting point about the store. In theory no component/controller should know the inner structure of the store, and so having `enketoStatus.saving` is therefore bad, however it does mean more boilerplate, and can potentially be confusing as `enketoStatus` is still used by the template.